build.sbt

```diff
@@ -16,8 +16,6 @@ ThisBuild / scalaVersion := scala213
 ThisBuild / githubWorkflowJavaVersions := Seq(JavaSpec.temurin("17"))
 ThisBuild / tlJdkRelease := Some(8)
 
-ThisBuild / testFrameworks += new TestFramework("munit.Framework")
```

Comment:
`munit` is supported out-of-the-box since sbt 1.5.0 - https://github.com/sbt/sbt/releases/tag/v1.5.0, https://github.com/sbt/sbt/pull/6335/files.